.github/resources/workspace-template.yaml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
Was this included because of the linter workflow changes?

Reply:
Yes.